Dockerfile

```diff
@@ -2,11 +2,12 @@ FROM python:3.6
 MAINTAINER Zach White <skullydazed@gmail.com>
```

Comment:
Per [this comment](https://github.com/qmk/qmk_api/pull/30#discussion_r221237450), line 1 should be changed to `FROM python:3.5`.
(Can't add a comment to line 1, so I'm commenting on this one instead.)